lib/results.go

```diff
@@ -156,6 +173,7 @@ func NewCSVEncoder(w io.Writer) Encoder {
 			strconv.FormatUint(r.Seq, 10),
 			r.Method,
 			r.URL,
+			string(header),
```

Comment:
This would break CSV parsing. We'd need to encode this differently. Perhaps Base64 encoding?